test/helper.rb

```diff
@@ -35,8 +35,39 @@ class Span
 end
 
 # Return a test tracer instance with a faux writer.
-def get_test_tracer
-  Datadog::Tracer.new(writer: FauxWriter.new)
+def get_test_tracer(options = {})
+  options = { writer: FauxWriter.new }.merge(options)
+  Datadog::Tracer.new(options).tap do |tracer|
```

Comment:
Do we need this defined in two places?

Reply:
Yes, they are different test suites, one Minitest, the other RSpec.

Reply:
ah, right, thanks.